### PR TITLE
Added the version 2.9.0 changelog

### PR DESCRIPTION
**Pull request by SachinSetiya**
# Description

Update the release notes with the 2.9.0 changelog

Please note that 2.9.0 is not merged into 2.9.x yet.
# Links

Fixes #[insert issue link, if any]

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by iroussos — APPROVED** [review]
Thank you @SachinSetiya, looks good to me! 

I've added a few comments for other reviewers to check, but I am approving on my side

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -14,6 +14,65 @@ Want to stay up-to-date with new releases? Subscribe to get
 [release notes email updates](https://www.timescale.com/signup/release-notes).
 </highlight>
 
+## What's new in TimescaleDB 2.9:
+This release includes these noteworthy features:
+*   Hierarchical Continuous Aggregates (aka Continuous Aggregate on top of another Continuous Aggregate)
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply timescaledb/overview/release-notes/index.md:19]: Continuous aggregates don't need to have initial capitalisation.

File: timescaledb/overview/release-notes/index.md [hunk 2]
```diff
@@ -14,6 +14,65 @@ Want to stay up-to-date with new releases? Subscribe to get
...
+*   Improve `time_bucket_gapfill` function to allow specifying the timezone to bucket
+*   Introduce fixed schedules for background jobs and the ability to check job errors.
+*   Use `alter_data_node()` to change the data node configuration. This function introduces the option to configure the availability of the data node.
+
+## 2.9.0 (2022-12-05)
+This release adds major new features since the 2.8.1 release.
+We deem it moderate priority for upgrading.
+
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply timescaledb/overview/release-notes/index.md:27]: I think that adding info here might take away from the important information, which is when readers need to be performing the update. We used to have standard wording for these, and they'd go into an admonition, but we seem to have lost it 😞

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -14,6 +14,65 @@ Want to stay up-to-date with new releases? Subscribe to get
...
+* #4670 Add timezone support to time_bucket_gapfill
+* #4678 Add interface for troubleshooting job failures
+* #4718 Add ability to merge chunks while compressing
+* #4786 Extend the now() optimization to also apply to CURRENT_TIMESTAMP
+* #4820 Support parameterized data node scans in joins
+* #4830 Add function to change configuration of a data nodes
+* #4966 Handle DML activity when datanode is not available
+* #4971 Add function to drop stale chunks on a datanode
+
+**Bugfixes**
```
Comment (Loquacity) [inline timescaledb/overview/release-notes/index.md:43]:
```suggestion
**Bug fixes**
```

File: timescaledb/overview/release-notes/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SachinSetiya) [reply timescaledb/overview/release-notes/index.md:19]: Edited

File: timescaledb/overview/release-notes/index.md (hunk 3 above)
Comment (SachinSetiya) [reply timescaledb/overview/release-notes/index.md:43]: Edited

**Review by Loquacity — APPROVED** [review]

**Comment by Loquacity** [comment]
Can we delete this pull request?

**Pull request closed without merging**